demisto_sdk/commands/validate/old_validate_readme.md

```diff
@@ -0,0 +1,154 @@
+## Validate
+
+Makes sure your content repository files are in order and have valid yml file scheme.
+
+**Notes**
+
+In order to run the README validator:
+- Node should be installed on you machine
+- The modules '@mdx-js/mdx', 'fs-extra', 'commander' should be installed in node-modules folder.
+    If not installed, the validator will print a warning with the relevant module that is missing.
+    please install it using "npm install *missing_module_name*"
+- 'DEMISTO_README_VALIDATION' environment variable should be set to True.
+    To set the environment variables, run the following shell commands:
+    export DEMISTO_README_VALIDATION=True
+
+In case of a private repo and an un-configured 'DEMISTO_SDK_GITHUB_TOKEN' or 'DEMISTO_SDK_GITLAB_TOKEN' validation of version bumps in files will be done with the local remote git branch.
```

Comment:
```suggestion
In case of a private repo and an un-configured 'DEMISTO_SDK_GITHUB_TOKEN' or 'DEMISTO_SDK_GITLAB_TOKEN', validation of version bumps in files will be done with the local remote git branch.
```